data/text/poke_dude.inc

```diff
@@ -1,154 +1,129 @@
-gUnknown_81C5F69::
+PokeDude_Text_SpeedierBattlerGoesFirst::
```

Comment:
`poke_dude` and `PokeDude` are inconsistent with `pokedude` and `Pokedude` used in other locations